dcos.yml

```diff
@@ -10,38 +10,67 @@
         timeout: 120
 
 - name: Collect DC/OS versions
-  hosts: all
+  hosts: all!win_agents
```

Comment:
we should run the setup step for `agents_windows` so please leave it to all

Reply:
done

Reply:
changed to agents_windows